README.md

```diff
@@ -60,6 +61,8 @@ See `monopacker --help` for details.
 You will need to know the builder or builders you want to build; `builder1 builder2` are used in the example here.
 
 ```shell
+gcloud auth login  # user credentials (every 24 hours)
+gcloud auth application-default login  # app credentails (every 24 hours)
```

Comment:
I learned recently that you can get rid of this second line if you add `--update-adc` to the first one, FYI.

Reply:
Aaaah. Awesome. :)

Readme updated.